README.md

```diff
@@ -1,2 +1,151 @@
+RFC are:
+
+   - A new feature that creates new API surface area
+   - The removal of features that already shipped
+   - The introduction of new idiomatic usage or conventions
+
+The RFC process is a great opportunity to get more eyeballs on your proposal
+before it becomes a part of a released version of Salt. Quite often, even
+proposals that seem "obvious" can be significantly improved once a wider group
+of interested people have a chance to weigh in.
+
+The RFC process can also be helpful to encourage discussions about a proposed
+feature as it is being designed, and incorporate important constraints into the
+design while it's easier to change, before the design has been fully
+implemented.
+
+Changes that do **NOT** require an RFC:
+
+  - Rephrasing, reorganizing or refactoring
+  - Bug fixes
+  - Addition or removal of warnings
+  - Additions only likely to be _noticed by_ other implementors-of-Salt,
+    invisible to users-of-Salt.
+
+## What the process is
+
+In short, to get a major feature added to Salt, one must submit the RFC via
+pull-request. After a comment period, the RFC will be either **Accepted** or
+**Rejected**. An **Accepted** RFC may then be implemented with the goal of
+eventual inclusion into Salt.
+
+## The RFC life-cycle
+The following is a more detailed explanation of the process.
+
+### 1. Proposal
+The RFC is proposed by submitting a pull request to this repo, by copying
+`0000-template.md` and modifying it. If the RFC pertains to any open issues,
+reference them in the **Salt Issue(s)** entry.
+
+When copying the file, do _not_ assign it a number. Simply name the file with a
+short description of the RFC (e.g. `subspace-transport.md`). Once the pull
+request has been opened, a SaltStack core engineer will assign the RFC a number
+and the RFC will enter the initial **Draft** status. At this time, the
+following changes can be made to the RFC file:
+
+1. Add the pull request number to the **RFC PR** entry at the top of the file
+2. Rename the file to include the assigned RFC number, then commit and push to
+   update the pull request
+    ```bash
+    $ git mv subspace-transport.md 0123-subspace-transport.md
+    $ git commit -am 'Assigned RFC number'
+    $ git push origin branchname
+    ```
+
+### 2. Discussion
+The pull request will remain open and serve as the comment thread for the RFC.
+The initial comment period will last no fewer than two (2) weeks, and may be
+extended as deemed necessary based on comment activity.
+
+Once the initial comment period has ended, the RFC will enter **Final Comment**
+status. One (1) week will be allowed for any further comments. At the end of
+the **Final Comment** period, a decision will be made on whether or not to
+accept the RFC. Acceptance requires approval from five (5) members of the core
+development team. As project creator, [Thomas
+Hatch](https://github.com/thatch45) will have a final veto on any RFC.
+
+### 3. Acceptance / Rejection
```

Comment:
Should rejected RFCs be put into their own folder so that they aren't intermingled with other RFCs?

Reply:
Well. Rejected RFC so far unmerged RFC. So it is enough to "close PR" and that's it. :wink:

Reply:
@DWSR We've discussed both sides of the coin. (also, rejected RFCs *will* be merged, so we can use them later to point to and say, "No, we're not doing that because it's already been proposed" - basically taking a page from the PEP process, rejected PEPs are still PEPs)

I believe we've talked about simply maintaining a .md file here in the repo with a list of "accepted" and "rejected" RFCs. Then one can see at a glance what RFCs even exist, just by scrolling through the file list here on GitHub. Even if we hit 1,000 RFCs, it still shouldn't be awful to scroll through.

What's your scenario scenario/thought behind sticking them in their own folder?

Reply:
Sticking rejected RFCs in their own folder makes them a little less visible than accepted RFCs, but maintaining a TOC is a good compromise, I think.

I would love to see a static site though

Reply:
Actually, rejected RFCs can be also a good material for future work and tryagains.